projects/observability/src/shared/graphql/model/metric/metric-timeseries.ts

```diff
@@ -1,4 +1,9 @@
 export interface MetricTimeseriesInterval {
-  value: number;
   timestamp: Date;
+  value: number;
+}
+
+export interface MetricTimeseriesBandInterval extends MetricTimeseriesInterval {
```

Comment:
Should the band include the baseline? I was thinking the baseline is treated as a regular series (with a different spec) and the band is the one with its own unique config.